cypress/integration/ui_spec.js

```diff
@@ -379,6 +380,72 @@ describe("UI", () => {
 					win.parleySettings.roomNumber = newAccountIdentification;
 				});
 
+				cy.wait("@createDevice");
+			});
+			it("should clear the messages when switching accounts", () => {
+				const parleyConfig = {roomNumber: "0W4qcE5aXoKq9OzvHxj2"};
+				const testMessage = `test message before switching room numbers ${Date.now()}`;
+
+				cy.visit("/", {
+					onBeforeLoad: (win) => {
+						// eslint-disable-next-line no-param-reassign
+						win.parleySettings = parleyConfig;
+					},
+				});
+
+				cy.get("[id=app]").as("app");
+
+				clickOnLauncher();
+				sendMessage(testMessage);
+				findMessage(testMessage); // Wait until the server received the new message
+
+				// Change the account identification
+				const newAccountIdentification = "0cce5bfcdbf07978b269";
+				cy.window().then((win) => {
+					// eslint-disable-next-line no-param-reassign
+					win.parleySettings.roomNumber = newAccountIdentification;
+				});
+
+				cy.get("@app")
+					.find("[class^=wrapper__]")
+					.should("be.visible")
+					.find("[class^=body__]")
+					.should("be.visible")
+					.should("not.contain", testMessage);
+			});
+		});
+		describe("xIrisIdentification", () => {
```

Comment:
Een describe is toch bedoelt om tests te groeperen. Maar dit is een groep met maar 1 test erin. Is het dan wel nuttig om er een groep omheen te zetten?

Reply:
Ja want dan zie je netjes voor welke setting de test is 
![image](https://user-images.githubusercontent.com/26112463/176126650-1b3b6121-07dc-47e1-b927-1bedb1cc4a2a.png)

Alles onder `parley config settings` is een config setting en heeft dus z'n eigen `describe()`